scripts/bootstrap/compile.sh

```diff
@@ -17,12 +17,13 @@
 # Script for building bazel from scratch without bazel
 
 if [ -d derived/jars ]; then
-  PROTOBUF_JARS=derived/jars
+  ADDITIONAL_JARS=derived/jars
 else
-  PROTOBUF_JARS="/usr/share/java/protobuf.jar /usr/share/java/protobuf-java-util.jar"
+  DISTRIBUTION=${DISTRIBUTION:-debian}
+  ADDITIONAL_JARS="$(grep -o "\".*\.jar\"" tools/distributions/${DISTRIBUTION}/${DISTRIBUTION}_java.BUILD | sed "s/\"//g" | sed "s/^/\/usr\/share\/java\//g")"
 fi
 PROTO_FILES=$(find third_party/remoteapis third_party/googleapis third_party/pprof src/main/protobuf src/main/java/com/google/devtools/build/lib/buildeventstream/proto src/main/java/com/google/devtools/build/skyframe src/main/java/com/google/devtools/build/lib/skyframe/proto src/main/java/com/google/devtools/build/lib/bazel/debug src/main/java/com/google/devtools/build/lib/starlarkdebug/proto -name "*.proto")
-LIBRARY_JARS=$(find $PROTOBUF_JARS third_party -name '*.jar' | grep -Fv JavaBuilder | grep -Fv third_party/guava | grep -Fv third_party/guava | grep -ve 'third_party/grpc/grpc.*jar' | tr "\n" " ")
+LIBRARY_JARS=$(find $ADDITIONAL_JARS third_party -name '*.jar' | grep -Fv JavaBuilder | grep -Fv third_party/guava | grep -Fv third_party/guava | grep -ve 'third_party/grpc/grpc.*jar' | tr "\n" " ")
```

Comment:
Can you add a comment here why we are excluding some jars?

Also see the duplicate guava that I mentioned in the other commit

Reply:
Actually, looks like we don't have to exclude them. JavaBuilder is no longer  checked into third_party. And we have only single version of grpc and guava checked in. But I'll fix them later we I try to replace them with system jars.

The duplicated guava is also fixed in previous PR, I'll make sure to rebase correctly.